src/modeler/AutoRest.Swagger/Validation/SummaryDescriptionValidation.cs

```diff
@@ -0,0 +1,46 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using AutoRest.Core.Logging;
+using AutoRest.Core.Properties;
+using AutoRest.Swagger.Model;
+using AutoRest.Swagger.Validation.Core;
+using System;
+using System.Collections.Generic;
+
+namespace AutoRest.Swagger.Validation
+{
+    public class SummaryDescriptionValidation: TypedRule<Operation>
+    {
+        /// <summary>
+        /// Id of the Rule.
+        /// </summary>
+        public override string Id => "M2023";
+
+        /// <summary>
+        /// Violation category of the Rule.
+        /// </summary>
+        public override ValidationCategory ValidationCategory => ValidationCategory.SDKViolation;
+
+        /// <summary>
+        /// The template message for this Rule. 
+        /// </summary>
+        /// <remarks>
+        /// This may contain placeholders '{0}' for parameterized messages.
+        /// </remarks>
+        public override string MessageTemplate => Resources.SummaryDescriptionVaidationError;
+
+        /// <summary>
+        /// The severity of this message (ie, debug/info/warning/error/fatal, etc)
+        /// </summary>
+        public override Category Severity => Category.Error;
+
+        public override IEnumerable<ValidationMessage> GetValidationMessages(Operation operation, RuleContext context)
+        {
+            if(operation.Description != null && operation.Summary != null && operation.Description.Trim().Equals(operation.Summary.Trim(), StringComparison.OrdinalIgnoreCase))
+            {
+                yield return new ValidationMessage(new FileObjectPath(context.File, context.Path), this, new object[0]);
```

Comment:
When you return FileObjectPath, the path where this violation occurs would point to the Operation node in this case, if you want it to report as the description node or the summary node you may have to do an AppendProperty like here https://github.com/Azure/autorest/blob/e1729aeda9c4ade2bb07dd8314707f4c1f5e5f00/src/modeler/AutoRest.Swagger/Validation/ListOperationNamingWarning.cs#L46